recipe/meta.yaml

```diff
@@ -5,12 +5,11 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/l/louvain/louvain-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/l/louvain/louvain-{{ version }}.tar.gz
   sha256: ce040bb0c5e349aead5a1e5e99738dcd9f2d10c22526d0633281b6ae230ee8d4
 
 build:
   number: 0
```

Comment:
Please bump the version number still.

Reply:
Done!  I forgot to add this.

Reply:
Great, thanks! Maybe you still need to push the commit? I don't see the change here yet.

Reply:
Whoops, I did forget - see the last commit!